config/production.json

```diff
@@ -0,0 +1,6 @@
+{
+  "env": "prod",
+  "basket": {
+    "apiUrl": "https://basket.mozilla.org/news"
+  }
```

Comment:
Should [log.format](https://github.com/mozilla/fxa-basket-proxy/pull/2/files#diff-f7bf2b665273dfa66ffa4ad7c0a52bb9R23) be changed to `heka` for production?